### PR TITLE
CloudNet 1.36.2.3

### PR DESCRIPTION
**Pull request by dayunet2009**
CloudNet
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/120990)

### CODE REVIEW

**Comment by dayunet2009** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
I believe the city name `Wuhan` is causing the [Policy 2.5](https://learn.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies#25-offensive-content) check, though speculating on the connection might invoke a controversy.

**Comment by stephengillie** [comment]
image

File: manifests/c/cloudnet/cloudnet/1.36.2.3/cloudnet.cloudnet.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿# yaml-language-server: =https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: cloudnet.cloudnet
+PackageVersion: 1.36.2.3
+InstallerLocale: zh-CN
+InstallerType: inno
+Scope: machine
+InstallerSwitches:
+  Custom: -norestart
+UpgradeBehavior: install
```
Comment (stephengillie) [inline manifests/c/cloudnet/cloudnet/1.36.2.3/cloudnet.cloudnet.installer.yaml:10]:
```suggestion
UpgradeBehavior: install
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by stephengillie** [comment]
Hi @dayunet2009,

This package installed normally, but needed a dependency to run.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**